extensions/libarchive/MergeContent.cpp

```diff
@@ -62,6 +62,11 @@ core::Property MergeContent::KeepPath(
   core::PropertyBuilder::createProperty("Keep Path")
   ->withDescription("If using the Zip or Tar Merge Format, specifies whether or not the FlowFiles' paths should be included in their entry")
   ->withDefaultValue(false)->build());
+core::Property MergeContent::AttributeStrategy(
+  core::PropertyBuilder::createProperty("Attribute Strategy")
+  ->withDescription("Determines which FlowFile attributes should be added to the bundle.")
```

Comment:
Can you please extend the description to the one used in the docs here?
http://nifi.apache.org/docs/nifi-docs/components/org.apache.nifi/nifi-standard-nar/1.11.4/org.apache.nifi.processors.standard.MergeContent/